db4-graph/src/lib.rs

```diff
@@ -68,6 +74,27 @@ impl AsRef<Path> for GraphDir {
     }
 }
 
+/// A simple transaction manager that currently just keeps track of transaction IDs.
+#[derive(Debug)]
+pub struct TransactionManager {
+    last_transaction_id: AtomicU64,
```

Comment:
This probably needs to be updated on replay

Reply:
I've added a `load` method to `TransactionManager` for now when we implement replay in the future.